torch/_export/passes/replace_set_grad_with_hop_pass.py

```diff
@@ -160,7 +160,58 @@ def _maybe_inline_or_replace_with_hop(node: torch.fx.Node):
     return gm
 
 
-def replace_set_grad_with_hop_pass(gm: torch.fx.GraphModule):
+def replace_set_grad_with_hop_pass(gm: torch.fx.GraphModule, graph_signature):
```

Comment:
maybe a type annotation?

Reply:
yeah adding a type annotation here is better.